.github/workflows/content-sources-actions.yml

```diff
@@ -122,8 +122,23 @@ jobs:
           docker run --network=host --rm -v ${PWD}:/local curlimages/curl  \
             curl --retry-all-errors --fail --retry-delay 10 --retry 32 --retry-max-time 240 http://localhost:8080/api/pulp/default/api/v3/repositories/rpm/rpm/  -u admin:password
           sleep 30
+      - name: start candlepin
+        uses: isbang/compose-action@v1.4.1
+        with:
+          compose-file: ./deployments/docker-compose.yml
+          services: |
+            candlepin
+          down-flags: --volumes
+        env:
+          CONTENT_DATABASE_PORT: 5434
```

Comment:
I want to point out that despite specifying the candlepin service, the db service is also being started (probably because the candlepin service is defined as being dependent on it). I set a different database port for the db so it doesn't interfere with the one already running.